README.md

```diff
@@ -820,16 +848,34 @@ Deploying `relp input` reading logs from remote rsyslog and `remote_files output
         outputs: [remote_files_output]
 ```
 
-### Port and SELinux
+### Port Managed by Firewall and SELinux Role
+
+When a port is specified in the logging role configuration,
+the firewall role is automatically included and the port
+is managed by the firewalld.
+
+The port is then configured by the selinux role and given
+an appropriate syslog selinux port type depending upon the
+associated tls value.
```

Comment:
tls value?

Reply:
Some inputs/outputs (e.g., relp) have a `tls` key in the dictionary like this.
```
- `tls`: If true, encrypt the connection with TLS. You must provide key/certificates and triplets {`ca_cert`, `cert`, `private_key`} and/or {`ca_cert_src`, `cert_src`, `private_key_src`}. Default to `true`.
 ```
I'm going to put ` around tls as follows. Do you think it's clearer?

859      associated `tls` value.

Reply:
ah, I see - then use TLS in all caps.